Code/Common/include/SimpleITK.h

```diff
@@ -72,6 +72,20 @@
 
 #include "sitkAdditionalProcedures.h"
 
+/* 
+ * An intentional design choice to have to explicitly include the Image
+ * operators and not get them by with the basic "SimpleITK.h".
+ * The addition of these convenience mathematical operators
+ * in C++ can be a source of confusion.
+ *
+ * Either:
+ *   1) explicity include "sitkImageOperators.h" (Recommended)
+ *   2) or define USE_SimpleITK_ImageOperators BEFORE including SimpleITK.h
+ */
+#ifdef USE_SimpleITK_ImageOperators
```

Comment:
Is this intended to be a CMake option, propagated to a configuration file?

The following a comparison of the proposed usage:

```
#define USE_SimpleITK_ImageOperators
#include "SimpleITK.h"
```
vs
```
#include "SimpleITK.h"
#include "sitkImageOperators.h"
```

I am not sure the `define` works so well. Perhaps this comment could just be added to a Doxygen `\file` block?